packages/manager/modules/vps/src/dashboard/migrate/migrate.html

```diff
@@ -0,0 +1,113 @@
+<h4 data-translate="vps_dashboard_migrate_title"></h4>
+<div class="w-25 mt-3 oui-select-picker">
+    <label class="oui-select-picker__label-container">
+        <span class="oui-select-picker__body">
+            <h6 data-translate="vps_dashboard_migrate_current_range_title"></h6>
+            <div class="oui-select-picker__section">
+                <ul class="list-unstyled">
+                    <li>
+                        <span
+                            data-translate="vps_dashboard_migrate_configuration_label_core"
+                        ></span>
+                        <span
+                            data-translate="vps_dashboard_migrate_configuration_vcore"
+                            data-translate-values="{ count: $ctrl.stateVps.vcore }"
+                        ></span>
+                    </li>
+                    <li>
+                        <span
+                            data-translate="vps_dashboard_migrate_configuration_label_ram"
+                        ></span>
+                        <span
+                            data-ng-bind=":: $ctrl.formatMemory($ctrl.stateVps.model.memory) | cucBytes:0:false:'GB'"
+                        ></span>
+                    </li>
+                    <li>
+                        <span
+                            data-translate="vps_dashboard_migrate_configuration_label_disk"
+                        ></span>
+                        <span
+                            data-ng-bind=":: $ctrl.stateVps.model.disk | cucBytes:0:false:'GB'"
+                        ></span>
+                    </li>
+                </ul>
+            </div>
+        </span>
+    </label>
+</div>
+
+<p class="mt-3">
+    <span data-translate="vps_dashboard_migrate_information_1"></span><br />
+    <span data-translate="vps_dashboard_migrate_information_2"></span>
+</p>
+
+<p class="mt-3">
+    <b data-translate="vps_dashboard_migrate_link_information"></b>
+    <br />
+    <a
+        data-ng-href="{{:: $ctrl.MIGRATE_FAQ_LINK }}"
+        target="_blank"
+        rel="noopener"
+    >
+        <span data-translate="vps_dashboard_migrate_faq_title"></span>
+        <span class="oui-icon oui-icon-external-link" aria-hidden="true"></span>
+    </a>
+</p>
+
+<oui-select-picker
+    class="w-25"
+    label="{{:: 'vps_dashboard_migrate_new_range_title' | translate }}"
+    model="$ctrl.selectedPlan"
+>
+    <oui-select-picker-section>
+        <ul class="list-unstyled">
```

Comment:
I'm wondering if a `dl` could be better here

Reply:
Then i will have to use custom `classes` to make sure they are `horizontally` aligned. I think `ul` and `li` is better here as we dont have to add class to every row.